redbot/core/bot.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, *args, cli_flags=None, bot_dir: Path = Path.cwd(), **kwargs):
             invite_commands_scope=False,
             disabled_commands=[],
             disabled_command_msg="That command is disabled.",
+            invoke_error_msg="`Error in command {command}. Check your console or logs for details.`",
```

Comment:
I think we should have `None` as the default and use the default message if config value is `None`. This way, we can continue to support translations in the default message *and* we can even only show the "Check your console or logs for details" if the command invoker was a bot owner, with the same check as in trivia cog.
```suggestion
            invoke_error_msg=None,
```